projects/6DofPose/demo.py

```diff
@@ -0,0 +1,116 @@
+import os
+import numpy as np
+from utils import *
+import argparse
+import torch
+import matplotlib.pyplot as plt
+
+import mmengine
+import mmcv
+
+from mmengine.registry import init_default_scope
+from mmpose.apis import inference_topdown
+from mmpose.apis import init_model as init_pose_estimator
+from mmpose.structures import merge_data_samples
+from mmpose.evaluation.functional import nms
+
+from mmdet.apis import inference_detector, init_detector
+
+
+def predict(image_path):
+    device = torch.device('cuda:0' if torch.cuda.is_available() else 'cpu')
+    
+    detector = init_detector(
+        '/home/liuyoufu/code/mmpose-openmmlab/mmpose/work_dirs/rtmdet_tiny_ape/rtmdet_tiny_ape.py',
+        '/home/liuyoufu/code/mmpose-openmmlab/mmpose/work_dirs/rtmdet_tiny_ape/best_coco_bbox_mAP_epoch_90.pth',
+        device=device)
+    
+    pose_estimator = init_pose_estimator(
+        '/home/liuyoufu/code/mmpose-openmmlab/mmpose/work_dirs/rtmpose-s_ape/rtmpose-s_ape.py',
+        '/home/liuyoufu/code/mmpose-openmmlab/mmpose/work_dirs/rtmpose-s_ape/best_PCK_epoch_240.pth',
```

Comment:
Would you like to share the config files and model checkpoints so that we can play this demo?

Reply:
Thanks for your review, I will gradually improve this pull request.